src/components/welcome/MyAccounts/styles.module.css

```diff
@@ -44,6 +44,12 @@
   background-color: var(--color-background-light) !important;
 }
 
+.currentListItem.multiListItem {
+  border-left-width: 1px;
```

Comment:
I think you don't need this first line, since the next one overwrites this one 🤔

Reply:
There is the single account component which uses only `currentListItem` without the `multiListItem` css class. 
I think its needed there.

Reply:
@schmanu I mean, in the same selector, line 49, you overwrite it with `border: 1px solid var(--color-border-light)` 🤔

Reply:
Ah sorry I misunderstood. I thought you meant the entire block not just the line -.- 
You are right. I will remove it :)